### PR TITLE
Check basic mtGET_LEDGER messages

### PR DESCRIPTION
**Pull request by lwojtow**

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -101,4 +101,13 @@ The test index makes use of symbolic language in describing connection and messa
 
     <>
     -> ping message with random `sequence` number
-    <- pong response with the same `sequence` number
\ No newline at end of file
+    <- pong response with the same `sequence` number
+
+### ZG-CONFORMANCE-004
+    
+    The node responds with mtLEDGER_DATA for mtGET_LEDGER
```
Comment (niklaslong) [inline SPEC.md:108]:
```suggestion
    The node responds with mtLEDGER_DATA for mtGET_LEDGER:
    
```

File: SPEC.md [hunk 2]
```diff
@@ -101,4 +101,13 @@ The test index makes use of symbolic language in describing connection and messa
...
+    -> mtGET_LEDGER with iType == LiBase
+    <- mtLEDGER_DATA
```
Comment (niklaslong) [inline SPEC.md:110]:
```suggestion
    <- mtLEDGER_DATA
    
```

File: SPEC.md [hunk 3]
```diff
@@ -101,4 +101,13 @@ The test index makes use of symbolic language in describing connection and messa
...
+    or:
```
Comment (niklaslong) [inline SPEC.md:111]:
```suggestion
    or:
    
```

File: SPEC.md (hunk 1 above)
Comment (ljedrz) [reply SPEC.md:108]: this should probably be a dot, like in the earlier ones

File: src/tests/conformance/query/ledger.rs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    // Start synth node and connect to Ripple
+    let mut synth_node = SyntheticNode::start().await.unwrap();
+    synth_node.connect(node.addr()).await.unwrap();
+
+    // Send message
+    synth_node.unicast(node.addr(), payload).unwrap();
+
+    // Wait for 'mtLEDGER_DATA' response
+    let check = |m: &BinaryMessage| matches!(&m.payload, Payload::TmLedgerData(..));
```
Comment (niklaslong) [inline src/tests/conformance/query/ledger.rs:58]: Note: in future we should check real data here (could be another test if multiple variations need to be checked).

File: SPEC.md (hunk 1 above)
Comment (niklaslong) [reply SPEC.md:108]: Suggestion updated.

**Pull request merged**